src/renderer-backend-egl.cpp

```diff
@@ -139,6 +139,11 @@ class Target {
 
     ~Target()
     {
+        if (m_wl.wpeBridgeId && m_glib.socket) {
+            uint32_t message[] = { 0x43, m_wl.wpeBridgeId };
+            g_socket_send(m_glib.socket, reinterpret_cast<gchar*>(message), 2 * sizeof(uint32_t), nullptr, nullptr);
```

Comment:
How about using `sizeof (message)` instead of `2 * sizeof (uint32_t)` to let the compiler calculate the number of bytes to send here? Or even better: define a `MESSAGE_SIZE` constant.

Reply:
Same here, I'll fix it in a follow up patch on top of this one.